packages/eslint-plugin-lit-a11y/lib/rules/interactive-supports-focus.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @fileoverview
+ * @author open-wc
+ */
+
+const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer.js');
+const { eventHandlersByType } = require('../utils/eventsHandlers.js');
+const { getAttrVal } = require('../utils/getAttrVal.js');
+const { getTabIndex } = require('../utils/getTabIndex.js');
+const { isDisabledElement } = require('../utils/isDisabledElement.js');
+const { isHiddenFromScreenReader } = require('../utils/isHiddenFromScreenReader.js');
+const {
+  isInteractiveElement,
+  isNonInteractiveElement,
+} = require('../utils/isInteractiveElement.js');
+const { isInteractiveRole, isNonInteractiveRole } = require('../utils/isInteractiveRole.js');
+const { isHtmlTaggedTemplate } = require('../utils/isLitHtmlTemplate.js');
+const { isPresentationRole } = require('../utils/isPresentationRole.js');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        'Elements with an interactive role and interaction handlers (mouse or key press) must be focusable.',
+      category: 'Fill me in',
+      recommended: false,
+    },
+    messages: {
+      elementWithRoleMustBe: "Elements with the '{{role}}' interactive role must be {{type}}.",
+    },
+    fixable: null, // or "code" or "whitespace"
+    schema: [
+      // fill in your schema
```

Comment:
the schema for `tabbable` array should be in here, otherwise line#48 won't do anything

Reply:
Added! I don't know if it needs to be required or not?